content/en/includes/bc-orca-forcecacherefresh.md

```diff
@@ -0,0 +1,43 @@
+#### ManifestForceCacheRefreshTask removed from Orca
+
+When you upgrade to 2.23.0 or later, you might encounter the following error:
+
+```bash
+2021-01-29 23:57:19.691 ERROR 1 --- [    scheduler-2] c.netflix.spinnaker.q.redis.RedisQueue   : Failed to read message 8f072714f1df6dbf3af93a4f4fe4cae2, requeuing...
+com.fasterxml.jackson.databind.JsonMappingException: No task found for 'com.netflix.spinnaker.orca.clouddriver.tasks.manifest.ManifestForceCacheRefreshTask' (through reference chain: com.netflix.spinnaker.orca.q.RunTask["taskType"])
+```
+
+The `ManifestForceCacheRefreshTask` task is no longer supported by the orchestration service, Orca. Because of this, messages from prior failed executions may have gotten stuck in the message queue, which is stored in Redis or MySQL.
```

Comment:
It's not that it's not supported, it's that it's not needed. When we deployed manifest one of the generated sub-task was to wait for force cache refresh. It's no longer emitted (no change needed for the user). However, if a task was being executed or retried before the upgrade, it will keep showing up in the log as an exception.